.jenkins/pipelines/ci_integrations.groovy

```diff
@@ -0,0 +1,66 @@
+pipeline {
+    agent {
+        docker {
+            image 'testmnrp/dart-converter:2'
+            args '-u root:root'
+        }
+    }
+    options {
+        timeout(time: 2, unit: 'HOURS')
+    }
+    environment {
+        METRICS_PROJECT_ID = "jenkins_ci_integrations"
+    }
+    stages {
+
+        stage('Git clone') {
+            steps {
+                git branch: 'master', url: 'https://github.com/platform-platform/monorepo.git'
+            }
+        }
+
+        stage('Get Dependencies') {
+            steps {
+                dir('metrics/ci_integrations'){
+                    sh 'pub get'
+                }
+            }
+        }
+
+        stage('Run tests') {
+                    failFast true
+                    parallel {
+                        stage('Run Analyzer') {
+                            steps {
+                                dir('metrics/ci_integrations'){
+                                    sh 'dartanalyzer . --fatal-infos --fatal-warnings'
+                                }
+                            }
+                        }
+                        stage('Test coverage') {
+                            steps {
+                                dir('metrics/ci_integrations'){
+                                    sh 'pub run test_coverage --print-test-output --no-badge --port 9498'
+                                }
+                            }
+                        }
+                   }
+               }
+
+        stage('Convert coverage report') {
+            steps {
+                dir('metrics/ci_integrations'){
+                    sh 'coverage_converter lcov -i coverage/lcov.info -o coverage-summary.json'
+                }
+            }
+        }
+    }
+
+    post {
+        always {
+            archiveArtifacts artifacts: 'metrics/ci_integrations/coverage-summary.json', fingerprint: true
```

Comment:
I wonder, does the `build job` command will run if the `archiveArtifacts` fail due to the artifact file not found? Maybe we should add `allowEmptyArchive: true` to avoid the situation when the `sync_build_data ` job won't run since we should synchronize the build data even when there is no artifact for some reason (the build failed before running tests, for example).